ui/pages/settings/developer-options-tab/developer-options-tab.tsx

```diff
@@ -255,6 +272,40 @@ const DeveloperOptionsTab = () => {
     );
   };
 
+  const renderEnableConfirmationsRedesignToggle = () => {
+    return (
+      <Box
```

Comment:
Can we create a local `Toggle` component to remove all the duplication in each setting?

Reply:
done